src/Api/SecretManagerFeatures/Models/Request/SecretCreateRequestModel.cs

```diff
@@ -18,15 +18,29 @@ public class SecretCreateRequestModel
     [EncryptedString]
     public string Note { get; set; }
 
+    public Guid? ProjectId { get; set; }
+
     public Secret ToSecret(Guid organizationId)
     {
-        return new Secret()
+        var s = new Secret()
         {
             OrganizationId = organizationId,
             Key = Key,
             Value = Value,
             Note = Note,
             DeletedDate = null,
         };
+
```

Comment:
We should be able to reduce having to make local variables with something like this. 

```
  public Secret ToSecret(Guid organizationId)
    {
        return new Secret()
        {
            OrganizationId = organizationId,
            Key = Key,
            Value = Value,
            Note = Note,
            DeletedDate = null,
            Projects = ProjectId.HasValue ? new List<Project>() { new(){ Id = ProjectId.Value } } : null
        };
    }
```